src/SqlPersistence/Outbox/SqlOutboxFeature.cs

```diff
@@ -28,6 +39,18 @@ protected override void Setup(FeatureConfigurationContext context)
             var frequencyToRunCleanup = settings.GetOrDefault<TimeSpan?>(FrequencyToRunDeduplicationDataCleanup) ?? TimeSpan.FromMinutes(1);
             var timeToKeepDeduplicationData = settings.GetOrDefault<TimeSpan?>(TimeToKeepDeduplicationData) ?? TimeSpan.FromDays(7);
 
+            var diagnostics = new Dictionary<string, object>
+            {
+                { "CleanupDisabled", false },
+                {nameof(timeToKeepDeduplicationData), timeToKeepDeduplicationData },
```

Comment:
Maybe hardcode the name (`"TimeToKeepDeduplicationData"`)here so that if follows the `PascalCase` convention?